.circleci/config.yml

```diff
@@ -43,6 +43,7 @@ jobs:
               git clone git@github.com:plotly/dash-html-components.git
               git clone git@github.com:plotly/dash-table.git
               . venv/bin/activate
+              cd dash && git checkout clientside-2 && cd ..
```

Comment:
Note: You can also do add `-b clientside-2` in the git clone command above.

Reply:
non-⛔️ as this line will disappear when the companion PR merges... but we may benefit 🐎 from using a shallow clone `git clone -depth=1` on CI, in which case `-b <branch>` may be the required method.